analysis/applySelections.cpp

```diff
@@ -173,19 +217,40 @@ int main(int argc, char* argv[])
     outFilePaths.push_back(argv[7]); // LbL signal extraction
     outFilePaths.push_back(argv[8]); // QED signal extraction
   }
+
   if(argc == 5){
     configPath = argv[1];
     inFilePath = argv[2];
     outFilePaths.push_back(argv[3]);
     outFilePaths.push_back(argv[4]);
   }
+  
   if(argc == 4){
     configPath = argv[1];
     inFilePath = argv[2];
     outFilePaths.push_back(argv[3]);
   }
+
+    if(argc == 3){
+    std::string arg_1 = argv[1];
+    std::string arg_2 = argv[2];
+    if(arg_1 == "TauTau"){
+    vector<string> arg_list; 
+    std::ifstream file(arg_2);
+    std::string str; 
+    while (std::getline(file, str))
+    {
+        arg_list.push_back(str);
+    }
+    configPath = arg_list[0];
+    inFilePath = arg_list[1];
+    outFilePaths.push_back(arg_list[2]); // single mu
+    outFilePaths.push_back(arg_list[3]); // muon electron
+    outFilePaths.push_back(arg_list[4]); // dimuon 
+    }
+  }
```

Comment:
Fix indentation (please use the same as through the project, which is 1 indent = 2 spaces). Please use consistent styling for brackets: loops, statements etc. - same line, methods/functions - next line.